public/pages/DetectorsList/Components/ConfirmActionModals/ConfirmActionModals.tsx

```diff
@@ -43,98 +55,210 @@ export const ConfirmStartDetectorsModal = (
   props: ConfirmStartDetectorsModalProps
 ) => (
   <EuiOverlayMask>
-    <ConfirmModal
-      title="Are you sure you want to start the selected detectors?"
-      description=""
-      callout={
+    <EuiModal onClose={props.hideModal}>
+      <EuiModalHeader>
+        <EuiModalHeaderTitle>
+          {'Are you sure you want to start the selected detectors?'}&nbsp;
+        </EuiModalHeaderTitle>
+      </EuiModalHeader>
+      <EuiModalBody>
         <EuiCallOut
           title="The following detectors will begin initializing:"
           color="success"
           iconType="play"
+        ></EuiCallOut>
+        <div>{getNamesGrid(props.detectors)}</div>
+      </EuiModalBody>
+      <EuiModalFooter>
+        <EuiButtonEmpty data-test-subj="cancelButton" onClick={props.hideModal}>
+          Cancel
+        </EuiButtonEmpty>
+        <EuiButton
+          data-test-subj="confirmButton"
+          color="primary"
+          fill
+          onClick={() => {
+            props.onStartDetectors();
+            props.hideModal();
+          }}
         >
-          <div>
-            {props.detectors.map(detector => {
-              return (
-                <li key={detector.id}>
-                  <EuiLink
-                    href={`${PLUGIN_NAME}#/detectors/${detector.id}`}
-                    target="_blank"
-                  >
-                    {detector.name} <EuiIcon type="popout" size="s" />
-                  </EuiLink>
-                </li>
-              );
-            })}
-          </div>
-        </EuiCallOut>
-      }
-      confirmButtonText="Start detectors"
-      confirmButtonColor="primary"
-      onClose={props.hideModal}
-      onCancel={props.hideModal}
-      onConfirm={() => {
-        props.onStartDetectors();
-        props.hideModal();
-      }}
-    />
+          {'Start detectors'}
+        </EuiButton>
+      </EuiModalFooter>
+    </EuiModal>
   </EuiOverlayMask>
 );
 
 export const ConfirmStopDetectorsModal = (
   props: ConfirmStopDetectorsModalProps
 ) => (
   <EuiOverlayMask>
-    <ConfirmModal
-      title="Are you sure you want to stop the selected detectors?"
-      description=""
-      callout={
+    <EuiModal onClose={props.hideModal}>
+      <EuiModalHeader>
+        <EuiModalHeaderTitle>
+          {'Are you sure you want to stop the selected detectors?'}&nbsp;
```

Comment:
Do we have those wording reviewed by Ashwin?

Reply:
All wording has been approved by Carlos. Do I need Ashwin as well?

Reply:
just realized this is same as previous title, should be fine

Reply:
> All wording has been approved by Carlos. Do I need Ashwin as well?

For such short ones, it should be fine. nvm.